data/json/recipes/recipe_weapon.json

```diff
@@ -767,1090 +767,746 @@
-  "book_learn": [[ "textbook_mechanics", 3 ] , [ "manual_pistol", 4] , [ "manual_shotgun", 4] , [ "manual_smg", 4] , [ "manual_rifle", 4]],
-  "using": [ [ "soldering_standard", 10 ] ],
-  "qualities":[
-    {"id":"HAMMER_FINE","level":1},
-    {"id":"SCREW_FINE","level":1}
-  ],
-  "tools": [
-    [
-      [ "small_repairkit", 10 ],
-      [ "large_repairkit", 10 ]
-    ]
-  ],
-  "components": [
-    [
-      [ "scrap", 1 ]
-    ]
-  ]
-},{
-  "type" : "recipe",
-  "result": "pistol_stock",
-  "category": "CC_WEAPON",
-  "subcategory": "CSC_WEAPON_MODS",
-  "skill_used": "fabrication",
-  "skills_required": [[ "gun", 3 ]],
-  "difficulty": 6,
-  "time": 30000,
-  "autolearn": true,
-  "book_learn": [[ "manual_fabrication", 3 ] , [ "manual_pistol", 4]],
-  "qualities":[
-    {"id":"HAMMER_FINE","level":1},
-    {"id":"SAW_M_FINE","level":1},
-    {"id":"SCREW_FINE","level":1},
-    { "id" : "GLARE", "level" : 2 }
-  ],
-  "tools": [
-    [
-      [ "oxy_torch", 3 ],
-      [ "welder", 15 ],
-      [ "welder_crude", 25 ],
-      [ "toolset", 25 ],
-      [ "small_repairkit", 15 ],
-      [ "large_repairkit", 15 ]
-    ]
-  ],
-  "components": [
-    [
-      [ "scrap", 6 ],
-      [ "pipe", 1 ],
-      [ "steel_chunk", 3 ]
-    ],
-    [
-      [ "plastic_chunk", 3 ]
-    ]
-  ]
-},{
-  "type" : "recipe",
-  "result": "rail_laser_sight",
-  "category": "CC_WEAPON",
-  "subcategory": "CSC_WEAPON_MODS",
-  "skill_used": "electronics",
-  "difficulty": 4,
-  "time": 15000,
-  "book_learn": [[ "manual_electronics", 2 ] , [ "textbook_electronics", 2 ]],
-  "using": [ [ "soldering_standard", 10 ] ],
-  "qualities" : [
-    {"id":"SCREW_FINE","level":1}
-  ],
-  "components": [
-    [
-      [ "amplifier", 1 ]
-    ],
-    [
-      [ "power_supply", 1 ]
-    ],
-    [
-      [ "lens", 2 ]
-    ],
-    [
-      [ "battery", 100 ],
-      [ "plut_cell", 1 ]
-    ],
-    [
-      [ "scrap", 1 ],
-      [ "can_drink_unsealed", 1 ],
-      [ "can_food_unsealed", 1 ],
-      [ "canister_empty", 1 ],
-      [ "plastic_chunk", 1 ]
-    ],
-    [
-      [ "cable", 5 ]
-    ]
-  ]
-},{
-  "type" : "recipe",
-  "result": "recoil_stock",
-  "category": "CC_WEAPON",
-  "subcategory": "CSC_WEAPON_MODS",
-  "skill_used": "fabrication",
-  "skills_required": [[ "gun", 3 ]],
-  "difficulty": 6,
-  "time": 30000,
-  "autolearn": true,
-  "book_learn": [[ "manual_fabrication", 3 ] , [ "manual_shotgun", 4] , [ "manual_smg", 4] , [ "manual_rifle", 4] , [ "manual_launcher", 4]],
-  "qualities":[
-    {"id":"HAMMER_FINE","level":1},
-    {"id":"SAW_M_FINE","level":1},
-    {"id":"SCREW_FINE","level":1},
-    { "id" : "GLARE", "level" : 2 }
-  ],
-  "tools": [
-    [
-      [ "oxy_torch", 3 ],
-      [ "welder", 15 ],
-      [ "welder_crude", 25 ],
-      [ "toolset", 25 ],
-      [ "small_repairkit", 15 ],
-      [ "large_repairkit", 15 ]
-    ]
-  ],
-  "components": [
-    [
-      [ "scrap", 6 ],
-      [ "pipe", 1 ],
-      [ "steel_chunk", 2 ]
+  },  {
```

Comment:
Linter did this?

Reply:
No. The linter doesn't support linting small sections of json without [] so each chunk had to be linted as a mini-file. I added the whitespace while adding and trimming [] characters and aligning the chunks.